sqrt/__init__.py

```diff
@@ -48,6 +49,8 @@ def get_data_stream(iterable):
 
 
 def main(save_to, num_batches, continue_=False):
+    if continue_:
+        continue_training(save_to)
```

Comment:
Well, `continue_training` is a 4-line function, it shouldn't be a problem for a user to load a main loop manually and interact with it.